libs/wingc/src/type_check.rs

```diff
@@ -727,6 +735,7 @@ impl TypeRef {
 	pub fn is_capturable(&self) -> bool {
 		match **self {
 			Type::Resource(_) => true,
+			Type::Interface(_) => true,
```

Comment:
Why do we need this? It's not strictly correct.

Reply:
This is necessary so that the resources typed as interfaces can be used inflight. Here's an example:

```js
bring cloud;

resource A {
  fn: cloud.IFunctionHandler;
  init(fn: cloud.IFunctionHandler) {
    this.fn = fn;
  }
  inflight do_stuff(message: str): str {
    this.fn.handle(message);
  }
}

resource Greeter impl cloud.IFunctionHandler {
  init() {}
  inflight handle(message: str): str {
    return "Hello ${message}";
  }
}

let fn = new A(new Greeter());
```

Perhaps we need some way to distinguish between interfaces on resources and interfaces on non-resources?

Reply:
OK, I've added a check. It's hacky since I couldn't see an easy way to check for type identity but it should be good enough for now